Sources/Maker/Maker+Edges.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Copyright © 2020 Rosberry. All rights reserved.
+//
+
+import UIKit
+
+extension Maker {
+
+    func _edges(top: Number? = nil, left: Number? = nil, bottom: Number? = nil, right: Number? = nil) {
+        apply(self._top, top)
+        apply(self._left, left)
+        apply(self._bottom, bottom)
+        apply(self._right, right)
+    }
+
+    func _edges(insets: UIEdgeInsets, sides: Sides = .all) {
+        sides.forEach { side in
+            switch side {
+            case .bottom:
+                _bottom(inset: insets.bottom)
+            case .left:
+                _left(inset: insets.left)
+            case .right:
+                _right(inset: insets.right)
+            case .top:
+                _top(inset: insets.top)
+            default:
+                return
+            }
+        }
+    }
+}
```

Comment:
I don't like the concept with `_` functions, because you still should copy implementations. What's about using protocols with default implementations?
```suggestion
public protocol Edges {

    @discardableResult func edges(top: Number?, left: Number?, bottom: Number?, right: Number?) -> Self
}

extension Maker: Edges {

    public func edges(top: Number? = nil, left: Number? = nil, bottom: Number? = nil, right: Number? = nil) -> Self {
        apply(self._top, top)
        apply(self._left, left)
        apply(self._bottom, bottom)
        apply(self._right, right)
        return self
    }
}
```